src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,194 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int MIN_LOGIN_LENGTH = 6;
+    private static final int MIN_PASSWORD_LENGTH = 6;
+    private static final int MIN_AGE = 18;
+    private StorageDao storageDao;
+    private RegistrationService registrationService;
+    private User defaultUser;
+
+    @BeforeEach
+    void setUp() {
+        storageDao = new StorageDaoImpl();
+        registrationService = new RegistrationServiceImpl();
+        defaultUser = new User();
+        defaultUser.setLogin("testLogin");
+        defaultUser.setPassword("testPassword");
+        defaultUser.setAge(33);
+
+    }
+
+    @Test
+    void register_ReturnValidUser_Ok() {
+        registrationService.register(defaultUser);
+
+        User result = storageDao.get(defaultUser.getLogin());
+        assertNotNull(storageDao.get(defaultUser.getLogin()), "User should be added to storage");
+        assertEquals(defaultUser, result, "Returned user should be the same as user added");
+    }
```

Comment:
Ive done this in case if MIN requirements for login, password or age will be changed.
If so, default user login length, password length or age can be wrong.
That's why I did this tests.
Theoretically I could just add @Test for default user.